### ISSUE TITLE
REE data

### ISSUE THREAD

**Issue by jerellejesse** [jerellejesse/HCR-shiny#2]
% REE for SSB and F do not look similar to report. Will investigate

**Comment by mdmazur** [comment]
I noticed this too. I think one issue is when there are two year updates, the code needs to be different for calculating REE than when there are one year updates. 
With one year updates:
`for (k in 1:length(sims)){
  load(sims[k])
  Ftrue<-omvalGlobal[[1]]$F_full[169:190]
  for (i in seq(170,190,1)){
    Fest<-omvalGlobal[[1]]$Fest[i,]
    Fest<-na.omit(Fest)
    Fest<-tail(Fest,1)
    Catchsim[(i-169),k]<-((Fest-Ftrue[i-169])/Ftrue[i-169])*100
  }
}`

With two year updates: 
`for (k in 1:length(sims)){
  load(sims[k])
  Ftrue<-omvalGlobal[[1]]$F_full[169:190]
  for (i in seq(170,190,2)){
    Fest<-omvalGlobal[[1]]$Fest[i,]
    Fest<-na.omit(Fest)
    Fest<-tail(Fest,1)
    Catchsim[(i-169),k]<-((Fest-Ftrue[i-169])/Ftrue[i-169])*100
  }
}`

**Comment by jerellejesse** [comment]
Updated data- some look better, but there are still some weird inconsistencies from the report that I may need help with

**Comment by mdmazur** [comment]
Which scenarios are inconsistent?

**Comment by jerellejesse** [comment]
I seems like a lot of them have the right pattern, but the magnitude is different or there is a spike. For example, this is the base case overfished
![image](https://user-images.githubusercontent.com/78380893/132393616-01bbca81-daad-4d7f-9ff0-a48713878c1e.png)
compared to the report
![image](https://user-images.githubusercontent.com/78380893/132393724-becf4c01-0554-4571-843b-d113dec5013b.png)

**Comment by mdmazur** [comment]
The most recent commit on the main branch has the correct code for the REE calculations. It's for REE for SSB, but you can apply the same structure to F.

**Comment by jerellejesse** [comment]
Thanks I will try it. I thought I did this separating out scenarios 21-24 as the annual updates and then merging them back together, but maybe I messed it up. I'll let you know.

**Comment by jerellejesse** [comment]
When I compared our code I realized that I had the years messed up that I was using in the calculations. Sorry about that, the years confuse me a little for these. I have updated the %REE for F to use the correct years now too and it looks all better! The y axis is a little squished for the recruitment misspecification and the base case not overfished, but I am seeing if I can do anything about that now.
